apps/betterangels-backend/notes/tests/test_queries.py

```diff
@@ -191,6 +195,45 @@ def test_notes_paginated_query(self) -> None:
         note_differences = DeepDiff(self.note, notes[0], ignore_order=True)
         self.assertFalse(note_differences)
 
+    def test_interaction_authors(self) -> None:
+        query = """
+            query ViewInteractionAuthors {
+                interactionAuthors {
+                    totalCount
+                    results {
+                        id
+                        firstName
+                        lastName
+                        middleName
+                    }
+                }
+            }
+        """
+
+        test_user_map = {
+            "test_new_interaction_author": baker.make(User, username=f"test_new_interaction_author_{uuid.uuid4()}")
+        }
+
+        test_new_interaction_author = test_user_map["test_new_interaction_author"]
+        test_new_interaction_author.first_name = "Wanda"
+        test_new_interaction_author.last_name = "Maximoff"
+        test_new_interaction_author.middle_name = "J."
+        test_new_interaction_author.save()
+
+        caseworker_permission_group_template = PermissionGroupTemplate.objects.get(name="Caseworker")
+        perm_group = permission_group_recipe.make(template=caseworker_permission_group_template)
+        perm_group.organization.add_user(test_new_interaction_author)
```

Comment:
you don't need to create a template here. `get_outreach_authorized_users` filters on `PermissionGroup` name, so you just need a permission group with the right name
```suggestion
        perm_group = permission_group_recipe.make(name=GroupTemplateNames.CASEWORKER)
        perm_group.organization.add_user(case_manager)
```